### PR TITLE
feat: review and update all info boxes

### LINKED ISSUE #1349: [Task]: Review and update all info boxes
### Task description

TBD

### PR DESCRIPTION
**Pull request by panteleymonchuk**
# Description of change

Closes #1349 

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] My code follows the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by brancoder — CHANGES_REQUESTED** [review]
We  should add some spacing between buttons when there are miltiple links:

image

File: client/src/assets/modals/nova/address/native-token.json
[the remark this answers is not in the document]
Comment (brancoder) [reply client/src/assets/modals/nova/address/native-token.json:1]: what i meant by this is that we can switch the links for Read more and Native tokens. wdyt?

File: client/src/assets/modals/nova/block/main-info.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "title": "What is block",
+    "description": "<p>In IOTA 2.0, a block is a fundamental unit within the Tangle, which is the underlying data structure of the network, organized as a directed acyclic graph (DAG). Each block in IOTA 2.0 carries a payload and is linked to previous blocks through references, embodying the network's transactions and data processing mechanisms.</p>",
+    "links": [
+        {
+            "label": "Read more",
+            "href": "https://wiki.iota.org/learn/protocols/iota2.0/core-concepts/data-structures#the-blocks",
+            "isExternal": true
+        },
+        {
+            "label": "Block Creation",
```
[the remark this answers is not in the document]
Comment (brancoder) [reply client/src/assets/modals/nova/block/main-info.json:11]: @begonaalvarezd wdyt?

File: client/src/assets/modals/nova/output/main-header.json
[the remark this answers is not in the document]
Comment (brancoder) [reply client/src/assets/modals/nova/output/main-header.json:1]: in the last commit looks like the search info got pasted in to the `output/main-header.json`.
for `output/main-header.json` we can use the info in my previous comment :arrow_up:

File: client/src/assets/modals/nova/search/main-header.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "title": "Search Criteria",
+    "description": "<p>All hex searches work with and without 0x prefix.<br />The following formats are supported:</p><ol><li><b>Blocks</b> 64 Hex characters</li><li><b>Block using Transaction Id</b> 64 Hex characters</li><li><b>Addresses</b> Bech32 Format</li><li><b>Nft Address</b> 66 Hex characters or Bech32 Format</li><li><b>Outputs</b> 68 Hex characters</li><li><b>SlotIndex</b> a unique index number</li><li><b>Slot Commitment ID</b> the identifier of the _slot_ this block commits to.</li><li><b>Epoch</b> a unique index number</li><li><b>Token Id</b> 76 Hex characters</li><li><b>NFT Id</b> 64 Hex characters</li></ol>"
```
[the remark this answers is not in the document]
Comment (brancoder) [reply client/src/assets/modals/nova/search/main-header.json:3]: Search criteria got pasted to output/main-header.json but it still has some issues in it like duplicated nft id

**Review by brancoder — APPROVED** [review]
nice work! LGTM :confetti_ball:

File: client/src/assets/modals/nova/block/main-info.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+    "title": "What is block",
```
[the remark this answers is not in the document]
Comment (begonaalvarezd) [reply client/src/assets/modals/nova/block/main-info.json:2]: Looking at the usage of this, im not sure why we even have block in the transaction metadat when the information is a bit mixed, we could actually remove this info
image

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]

**Review by begonaalvarezd — APPROVED** [review]

**Pull request merged**